crates/bevy_render/src/camera/projection.rs

```diff
@@ -234,6 +234,8 @@ impl CameraProjection for OrthographicProjection {
                 min_width,
                 min_height,
             } => {
+                // Compare Pixels of current width and minimal height and Pixels of minimal width with current height.
+                // Then use bigger (min_heigth when true) as what it referres to (height when true) and calculate rest so it can't get under minimum.
```

Comment:
```suggestion
                // Then use bigger (min_height when true) as what it refers to (height when true) and calculate rest so it can't get under minimum.
```